packages/newspringchurchapp/src/prayer/index.js

```diff
@@ -241,63 +253,82 @@ class PrayerMenu extends PureComponent {
 
   render() {
     return (
-      <SafeAreaView>
-        {this.state.showAddPrayerCard ? (
-          <StyledAddPrayerContainer>
-            <AddPrayerCardConnected
-              description={
-                'Take a moment to send a prayer request that your NewSpring Church family can pray for.'
-              }
-              {...this.props}
-            />
-          </StyledAddPrayerContainer>
-        ) : null}
-        {!this.state.showAddPrayerCard ? (
-          <StyledContainer>
-            <NSIcon
-              onPress={() => {
-                this.setState({
-                  showAddPrayerCard: true,
-                  prayerMenuItemSelected: 1,
-                });
-              }}
-              name={'arrow-up'}
-              fill={this.props.tint}
-              size={24}
-            />
-            <StyledButtonLink
-              onPress={() => {
-                this.setState({
-                  showAddPrayerCard: true,
-                  prayerMenuItemSelected: 1,
-                });
-              }}
-            >
-              Add your prayer
-            </StyledButtonLink>
-          </StyledContainer>
-        ) : null}
-        <RowHeader>
-          <H3>Pray for Others</H3>
-        </RowHeader>
-        <TabView
-          initialLayout={{
-            height: Dimensions.get('window').height,
-            width: Dimensions.get('window').width,
-          }}
-          navigationState={{ ...this.state }}
-          renderScene={SceneMap({
-            saved: this.tabRoute(1),
-            church: this.tabRoute(2),
-            campus: this.tabRoute(3),
-            community: this.tabRoute(4),
-            prayers: this.tabRoute(5),
-          })}
-          renderTabBar={this.renderTabBar}
-          onIndexChange={this.handleIndexChange}
-          swipeEnabled={false}
-        />
-      </SafeAreaView>
+      <Animated.View
+        style={{
+          transform: [
+            {
+              translateY: this.state.animatedValue.interpolate({
+                inputRange: [0, 1],
+                outputRange: [0, -(350 * this.state.showAddPrayerCard)],
+              }),
+            },
+          ],
+        }}
+      >
+        <SafeAreaView>
+          {this.state.showAddPrayerCard ? (
+            <StyledAddPrayerContainer>
+              <AddPrayerCardConnected
+                description={
+                  'Take a moment to send a prayer request that your NewSpring Church family can pray for.'
+                }
+                {...this.props}
+              />
+            </StyledAddPrayerContainer>
+          ) : (
+            <StyledContainer>
+              <NSIcon
+                onPress={() => {
```

Comment:
If I press the arrow and not the button link, bad things happen.
![Screenshot 2019-05-09 15 23 51](https://user-images.githubusercontent.com/3229463/57480753-95667680-726e-11e9-9e79-acab8506f72d.png)

Reply:
Good catch. Didn't even think about that.

Reply:
@richarddubay Fixed.